poppy/instrument.py

```diff
@@ -474,8 +474,12 @@ def _getOpticalSystem(self,fft_oversample=2, detector_oversample = None, fov_arc
 
 
         #---- apply pupil intensity and OPD to the optical model
-        optsys.addPupil(name='Entrance Pupil', optic=pupil_optic, transmission=full_pupil_path, opd=full_opd_path, opdunits='micron', rotation=self._rotation)
+        optsys.addPupil(name='Entrance Pupil', optic=pupil_optic, transmission=full_pupil_path, opd=full_opd_path, rotation=self._rotation)
```

Comment:
This would be a good case for incrementing the version number on the data files, then. 

I thought we had implemented some check for needing an updated version of that, but I don't see it anywhere in the code. There is a "version.txt" in the data directory but I don't see any code that checks that? I've got to be misremembering this somehow.